.gitignore

```diff
@@ -1 +1,2 @@
 /vendor/
+/.idea/
```

Comment:
This ignore should go into your global gitignore list, as it is not a direct artifact of the project, but rather a detail of your personal environment.